content/blog/2021-12-15-december-21-heartbeat.md

```diff
@@ -0,0 +1,269 @@
+parts, but lays out DVC's use for generating the DAG as well as logging metrics
+and CML to do the testing. We're looking forward to the next installments!
+
+![Heartbeat!](https://media.giphy.com/media/HYrBxW4xsPSP3wsUTk/giphy.gif)
+
+## Curious about Speaker Diarization?
+
+[The co-authored article entitled,](https://blogs.cisco.com/developer/speakerdiarization01)
+“Who Said That?” A Technical Intro to Speaker Diarization," by
+[**Dario Cazzani**](https://www.linkedin.com/in/dariocazzani/), and
+[**Alex Huang**](https://github.com/alhuang10), machine learning engineers at
+[Cisco,](https://www.cisco.com/) provides an introduction to the topic of
+Speaker Diarization, or who spoke when, in audio recordings. Their team's
+solution takes you through the fingerprinting of voices, clustering to assign
+speaker labels, creating the needed data pipeline, and the integration with
+Webex.
+
+In this process, the team derives benefit from using DVC to version data and
+models, as well as easily collaborate with each other and the transcription
+team. More info on this project can be found
+[in their repo.](https://github.com/CiscoDevNet/vo-id#train-the-vectorizer)
+
+![Speaker Diarization](/uploads/images/2021-12-15/Dario-Cazzani-2.png '=800')
+_Dario Cazzani and team's process for assinging speaker labels to audio files
+([Source link](https://blogs.cisco.com/developer/speakerdiarization01))_
+
+## Using DVC in Academic Research on a Compartmental Infectious Disease Model
+
+[**Matthew Segal**,](https://www.linkedin.com/in/matthew-segal-aa132093/)
+[in his post,](https://mattsegal.dev/devops-academic-research.html) "DevOps in
+Academic Research," reviews his work of applying some of the tried and true
+practices in DevOps to data science projects using a
+[Markov chain Monte Carlo](https://en.wikipedia.org/wiki/Markov_chain_Monte_Carlo)
+(MCMC) technique to create a model to simulate the spread of tuberculosis and
+later, as the pandemic erupted, COVID-19.
+
+The article covers mapping the workflow (see below), testing the codebase, smoke
+tests
+[(with a guide link),](https://mattsegal.dev/pytest-on-github-actions.html)
+contiunuous integration, and data management (where he recommends DVC).
+
+![Map Pipeline](/uploads/images/2021-12-15/matt-segal.png '=800') _Working to
+develop a pipeline
+([Source link](https://mattsegal.dev/devops-academic-research.html))_
+
+## Are you confused by how many MLOps tools there are?
+
+Well
+[Thoughtworks](https://www.thoughtworks.com/?utm_source=google-search&utm_medium=paid-media&utm_campaign=always-on-brand_2021-11&utm_term=thoughtworks&utm_content=RSAad1&gclid=Cj0KCQiA2NaNBhDvARIsAEw55hg2li5srltu8ppVsxLzcnv-pYWRmvnCk_jmljiC2ocyM4tc7EUEt9gaAoVWEALw_wcB)
+included DVC in its recent
+[Thoughtworks Guide to MLOps Platforms](https://www.thoughtworks.com/what-we-do/data-and-ai/cd4ml/guide-to-evaluating-mlops-platforms).
+While being included is great, things move so fast that they seemed to miss our
```

Comment:
Personally, I would say "they seem to have missed". Leaving it up to you though.

Reply:
Yep.  That's better.  Thanks!